config/locales/interface/input_elements/en_costs.yml

```diff
@@ -176,6 +176,17 @@ en:
         CO<sub>2</sub> capture. Costs are compared to current costs. Currently,
         this technology has only been applied experimentally at power plants. It is
         expected that costs will decrease in the future.\r\n<br/>"
+    costs_captured_biogenic_co2:
+      title: CO<sub>2</sub> removal price
+      short_description:
+      description: |
+        The CO<sub>2</sub> removal price is defined as the price that is received for
+        captured CO<sub>2</sub>. This slider sets the future CO<sub>2</sub> removal price (in €/tonne).
+        The start value is set as 0 €/tonne. <br/><br/>
+        Logically, <strong>the value of the CO<sub>2</sub> removal price should at least be
+        set equal to the CO<sub>2</sub> price</strong> (price for emitting CO<sub>2</sub>). If the
+        CO<sub>2</sub> removal price is set lower than the CO<sub>2</sub> price, it would be
```

Comment:
This description and should be updated to match the **captured biogenic CO2 price**. This also goes for the nl locales.